modules/globebrowsing/shaders/advanced_rings_fs.glsl

```diff
@@ -70,19 +70,19 @@ Fragment getFragment() {
     discard;
   }
 
-  vec4 colorBckwrd = texture(ringTextureBckwrd, texCoord);
-  vec4 colorFwrd = texture(ringTextureFwrd, texCoord);
-  vec4 colorMult = texture(ringTextureColor, texCoord);
-  vec4 transparency = texture(ringTextureTransparency, texCoord);
+  vec4 colorBckwrd = texture(textureBackwards, texCoord);
+  vec4 colorFwrd = texture(textureForwards, texCoord);
+  vec4 colorMult = texture(textureColor, texCoord);
+  float transparency = 1.0 - texture(textureTransparency, texCoord).r;
```

Comment:
```suggestion
  float transparency = 1.0 - texture(textureTransparency, texCoord).a;
```

Using the alpha channel for the single-channel textures to be in line with the other shaders